docs/index.md

```diff
@@ -24,8 +24,10 @@ These are additional lists of resources which may contain content not listed her
 
 - [rec-room Fandom Wiki Circuits V2 Page](https://rec-room.fandom.com/wiki/Circuits_V2)
 - [It's Time For YOU To Learn Circuits V2](https://www.youtube.com/watch?v=L4yvvoWdpWA)
+- [CV2 Chip Wiki](https://cv2.wiki)
 - [CV2 Chip Searcher](https://cv2browser.alyxisdysphoric.repl.co)
 
+
```

Comment:
Please delete the whitespace. Also, and I know you won't like this, put yours after the chip searcher for now. I think the chip searcher is slightly better r.n.

Reply:
How do I improve it lol